src/effects/fork.js

```diff
@@ -0,0 +1,11 @@
+import sg from '../sg';
+import createEffect from './createEffect';
+
+export const handleForkEffect = sgImpl => ([callable, ...args], parentEmitter, emitter) => {
+    const promise = sgImpl(callable, emitter)(...args);
```

Comment:
It doesn't check for callable to be a generator as done in [spawn](https://github.com/marmelab/sg/pull/8/files#diff-0e50efdcb840bb7943ea324e136720f6R6) ?